cluster/config-defaults.yaml

```diff
@@ -1202,6 +1202,13 @@ kube_janitor_enabled: "true"
 teapot_admission_controller_scheduling_controls_enabled: "false"
 teapot_admission_controller_scheduling_controls_default_architecture: "amd64"
 
+# master-node-autoscaler configuration
+
+# Ignore recommendations for downgrading the instance generation of the master nodes
+# This is required to persist the upgrade of instance generation for master nodes
+# in a cluster.
+master_node_autoscaler_instance_downgrade_disabled: "false"
```

Comment:
Note: MNA won't read this file so the default to false needs to be implemented in MNA.

Still fine to have it here as documentation for the field.

Reply:
Was implemented in the MNA as suggested. Thanks!